src/sound/SoundManager.js

```diff
@@ -830,7 +830,7 @@ Object.defineProperty(Phaser.SoundManager.prototype, "volume", {
                 {
                     if (this._sounds[i].usingAudioTag)
                     {
-                        this._sounds[i].volume = this._sounds[i].volume * value;
+                        this._sounds[i].volume = value;
```

Comment:
While I agree what we have currently doesn't work, this won't fix it either. All it will do is normalise all of the sounds to have the exact same volume, which isn't the end result we want.